build/LSan.supp

```diff
@@ -0,0 +1,27 @@
+#llvm memory hoggers
```

Comment:
We have put the equivalent file for valgrind in $BUILD_DIR/etc (and $ROOTSYS/etc) so that they can be re-used by users running the same type of checks.

Reply:
In principle, I agree, but valgrind doesn't support globbing on stack traces. The equivalent suppression file for valgrind would by **much** more verbose. If you know an automatic way to generate one, be my guest, but I won't do this manually.

Reply:
I am confused.  valgrind *does* support globbing .. but more important I am not asking for a new valgrind file but for **this** file (as is) to be in ./etc to be using by a user doing on its ROOT-using-library the ASAN analyze that you are introducing here.  (I guess it does depend on whether this file is read at run-time or compile-time, if it is the later then my advise is moot)

Reply:
@hageboeck This is marked as resolved but _I_ am still confused :).   

Is build/LSan.supp used only at build time or is it read at run-time>

If read at run-time, is build/LSan.supp of any use for external project?  Namely, if I external project build their project (and possibly ROOT) with the sanitizer on, should they be using build/LSan.supp ?

Thanks.

Reply:
Ok, got it. The suppression file can be used when an external project tries to use a ROOT with instrumentation. They will have to `LD_PRELOAD` or link the sanitizer runtimes, though. If that happens, all the leaks that are suppressed automatically in ROOT would come back.

Reply:
> LSAN_OPTIONS=suppressions=$ROOTSYS/etc/LSan.supp 

For symetry reason, consider using the name ```$ROOTSYS/etc/LSan-root.supp``` (and actually maybe ```lsan-root.supp```